### PR TITLE
TreeTopView

### PR DESCRIPTION
**Pull request by duhan9836**
You are given a pointer to the root of a binary tree. Print the top view of the binary tree.
Top view means when you look the tree from the top the nodes, what you will see will be called the top view of the tree.

### CODE REVIEW

File: treeTopView.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    def __str__(self):
+        return str(self.info)
+
+
+class BinarySearchTree:
+    def __init__(self):
+        self.root = None
+
+    def create(self, val):
```
Comment (caitao1985) [inline treeTopView.py:16]: better renamed to `insert`, can you implement a recursive version?

File: treeTopView.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                rightmost=levelright
+    print(" ".join(map(str,res)))
+
+
+def children(nodelvl):
+    ret = []
+    if nodelvl[0].left:
+        ret.append((nodelvl[0].left,nodelvl[1]-1))
+    if nodelvl[0].right:
+        ret.append((nodelvl[0].right,nodelvl[1]+1))
```
Comment (caitao1985) [inline treeTopView.py:75]: return

File: treeTopView.py (hunk 1 above)
Comment (duhan9836) [reply treeTopView.py:16]: the setup for class was given by the question. However, I'll think over the recursive implementation.

File: treeTopView.py (hunk 2 above)
Comment (duhan9836) [reply treeTopView.py:75]: thanks

**Pull request merged**